src/components/Tokens/TokenDetails/BalanceSummary.tsx

```diff
@@ -9,15 +9,14 @@ import styled, { useTheme } from 'styled-components/macro'
 import NetworkBalance from './NetworkBalance'
 
 const BalancesCard = styled.div`
-  width: 284px;
+  width: 320px;
```

Comment:
think you can make a container that wraps the balances and widget, set the width to 320px, then do width 100% for all items inside

Reply:
is that necessary tho

Reply:
bc i think widget you specify the height

Reply:
meh ok ill change